src/test/common/page/national-insurance-number.js

```diff
@@ -9,9 +9,9 @@ const PAGE_TITLES = {
 }
 
 /**
- * Page object for EnterNino page where the national insurance number is entered.
+ * Page object for NationalInsuranceNumber page where the national insurance number is entered.
```

Comment:
How about just: `Page object for the page where the national insurance number is entered.`